Libraries/Image/assetPathUtils.js

```diff
@@ -10,7 +10,7 @@
 
 'use strict';
 
-import type {PackagerAsset} from '../../Libraries/Image/AssetRegistry';
```

Comment:
Not sure if moving this utility and making it specific to Image makes sense. This is for all assets. Maybe this should be a separate util.

Reply:
Again, not an issue that we leave `local-cli`. However, this is only used in `Image` module on the React Native side.